_posts/2018-06-11-2018-foreman-survey-analysis.md

```diff
@@ -0,0 +1,406 @@
+author: Greg Sutcliffe
+tags:
+- usage
+- foreman
+- events
+- plugins
+- community
+- survey
+- API
+- virtualization
+- puppet
+- katello
+- pulp
+- content
+- use cases
+---
+
+As with previous years, we ran a Foreman Community Survey in order to give you
+all the opportunity to tell us how we're doing - where it's good, and where
+it's bad. That survey closed a while ago, and I'm here to show you the results.
+
+<!--more-->
+
+Firstly - **thank you** to all those who filled out the survey. We kept the same
+multi-page format since it seems to work, and even without prize incentives, we
+got over 160 responses! You're all legends :)
+
+If you've seen the previous community survey analysis posts, you'll note the
+style is a bit different this year. The two main changes are (a) lots of
+comparisons to the 2017 data, and (b) [no pie charts][evil-pies], since bar
+charts are generally better for comparisons between parts (which is mostly what
+we're doing). That's because I'm using R and R-Markdown to write the report with
+the code embedded, and you can find the RMarkdown [here][gitlab] if you want to
+check my working.
+
+Unless otherwise noted, the scales on these graphs are all percentages - I've
+removed the y-axis labels as it saves space and would be very repetitive.
+
+I appreciate this report is pretty long. If you're short on time, you can jump
+straight to the conclusion (or other sections) using these handy links:
+
+## Contents
+
+* [Intro](/2018/06/2018-foreman-survey-analysis.html)
+* [Page 1 - Community and Core](/2018/06/2018-foreman-survey-analysis.html#page1)
+* [Page 2 - Plugins, Provisioning, Monitoring](/2018/06/2018-foreman-survey-analysis.html#page2)
+* [Page 3 - Smart Proxy & Content](/2018/06/2018-foreman-survey-analysis.html#page3)
+* [Page 4 - Development & Contributing](/2018/06/2018-foreman-survey-analysis.html#page4)
+* [Conclusions](/2018/06/2018-foreman-survey-analysis.html#conclusions)
+
+The same page-by-page analysis from last year still works, so let's get to it
+with:
+
+## <a name="page1"></a>Community Metrics & Core
+
+Let's start with some data about the community itself. Here we see the "age" of
+users (as in, how long a user has been participating in the community),
+location, and some data about how quickly people upgrade:
+
+![](/static/images/blog_images/2018-survey-images/community-1.svg)<!-- -->![](/static/images/blog_images/2018-survey-images/community-2.svg)<!-- -->
+
+The geography data isn't too interesting - we see pretty much the same
+distribution as last year. Would be great to raise that "N America" bar though...
+
+The "age" chart is more concerning. Bear in mind that we're one year on from the
+2017 data (obviously), and we see an increase in the 2-3 year parts, and a
+corresponding drop in the < 1 year part. That implies that either (a) we're not
+doing a good job of bringing new users to the community, or (b) the new users
+weren't aware of the survey. Most likely it's a mix of both.
```

Comment:
was there a different approach to ask new users to fill in the survey this year? if not, I don't get why you imply it could be caused by awareness

Reply:
Not really, I just mean that newer users may be less aware of our communication channels, and thus missed the announcements. That was true last year too, but if (for some reason) that effect varied, it would show here.